src/lib/comms/include/sol-netctl.h

```diff
@@ -0,0 +1,581 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2015 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <sol-network.h>
+#include <sol-buffer.h>
+#include <sol-vector.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @struct sol_connman_service
+ *
+ * @brief service struct
+ */
+struct sol_connman_service;
```

Comment:
you must rename all `sol_connman` to `sol_netctl` in public headers and functions.